components/menu/menu.tsx

```diff
@@ -94,10 +94,12 @@ export function Menu(props: Props) {
     function handleMenuKeyDown(event: KeyboardEvent<HTMLDivElement>) {
         if (event.key === 'Enter') {
             const target = event.target as HTMLElement;
-            const ariaHasPopup = target?.getAttribute('aria-haspopup') === 'true';
+            const ariaHasPopupAttribute = target?.getAttribute('aria-haspopup') !== null ?? false;
```

Comment:
Does this still work if `aria-haspopup="false"` in the HTML?

Also, I don't think these need the `?? false` because the `!==` should always output a boolean value

Reply:
the default value of `aria-haspopup` is false, and should be omitted when not in use.

Changed the `??`